query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/PinotBasedRequestHandler.java

```diff
@@ -139,13 +143,12 @@ public void init(String name, Config config) {
    * Returns a QueryCost that is an indication of whether the given query can be handled by this
    * handler and if so, how costly is it to handle that query.
    *
-   * A query can usually be handled by Pinot handler if the Pinot view of this handler has all the
-   * columns that are referenced in the incoming query. If the Pinot view is a filtered view on
+   * <p>A query can usually be handled by Pinot handler if the Pinot view of this handler has all
```

Comment:
Generally tried to keep format-only changes out of this PR to prevent it from exploding, looks like I missed this one file.